lib/msf/core/modules/loader/executable.rb

```diff
@@ -82,6 +82,11 @@ def read_module_content(parent_path, type, module_reference_name)
       load_error(full_path, Errno::ENOENT.new)
       return ''
     end
-    Msf::Modules::External::Shim.generate(full_path)
+    begin
+      Msf::Modules::External::Shim.generate(full_path)
+    rescue ::Exception => e
+      elog "Unable to load module #{full_path} #{e.class} #{e}"
+      ''
```

Comment:
@acammack-r7 aren't `load_error` messages also printed out to the terminal when msfconsole starts? Is it a real bug that the python module can't be loaded or should this error only be shown in framework.log?
https://github.com/rapid7/metasploit-framework/blob/cac890a797d0d770260074dfe703eb5cfb63bd46/lib/msf/core/modules/loader/base.rb#L400